controller/translationcontroller.php

```diff
@@ -95,18 +95,18 @@ public function getLanguageStrings() {
 			'revision.restored' => $this->trans->t('Revision restored'),
 
 			// js/app/controllers/settings.js
-			'bookmarklet.text' => $this->trans->t('Save in passman'),
+			'bookmarklet.text' => $this->trans->t('Save in Passman'),
```

Comment:
Bookmarklet is [correct](https://en.wikipedia.org/wiki/Bookmarklet) english, please revert..